esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,100 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
+===================================================
+
+.. include:: ../common.txt
+
+**Please note:** Before you follow these steps to add your recipe, you must be
+able to successfully run it with ESMValtool without error.
+
+* Run ESMValTool locally with your recipe, make sure to take note of the memory
+  and time expenditure provided by the terminal output::
+
+    esmvaltool run <your_recipe_name.yml>
+
+* Run the same recipe on JASMIN, again taking note of the memory and time
+  expenditure printed to the terminal.
+
+* Stop any running recipe_test_workflow workflows::
+
+    cylc stop "a_running_recipe_test_workflow"
+
+* Add the recipe to the ``[task parameters]`` section of the |RTW| ``flow.cylc``
+  workflow file (make sure to place it within the "fast" or "medium" category
+  depending on how long it took to run):
+
+* Run the workflow. The process task for the new recipe should succeed, but its
```

Comment:
My recipe process task failed with this error: `slurmstepd: error: Exceeded step memory limit at some point.`

When run locally the recipe required:

```
2024-05-28 08:09:40,810 UTC [127076] INFO    Time for running the recipe was: 0:06:52.260589
2024-05-28 08:09:41,170 UTC [127076] INFO    Maximum memory used (estimate): 2.5 GB
```

Reply:
The latest instructions require that the duration and memory usage are taken from a run [on the compute server you use at your site](https://github.com/ESMValGroup/ESMValTool/blob/d236a324278f75c2cbb41d133e88f16dd18c72f7/esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst?plain=1#L6C1-L11C1), so the process task should not fail for this reason 👍 I will resolve this on @chrisbillowsMO's behalf, as discussed 😊